app/components/Free.js

```diff
@@ -0,0 +1,18 @@
+const React = require('react');
+const { defineSvg } = require('../defineSvg');
+
+module.exports = defineSvg({
+  className: 'tilt-icon-tag-free',
+
+  render() {
+    return (<svg className={this.getClassNames()} viewBox="0 0 64 64">
+      <title>Free</title>
+      <g stroke="none" strokeWidth="1" fill="none" fillRule="evenodd">
```

Comment:
these styles are best in CSS - see similar other recent PRs for best practices around this.  https://github.com/Crowdtilt/tilt-images/pull/35/files is a decent prototype